frontend/src/WelcomeModal/WelcomeModal.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import Dialog from "@material-ui/core/Dialog";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import MULogo from "MULogo/MULogo";
+import { makeStyles, useTheme, Theme } from "@material-ui/core/styles";
+
+interface IWelcomeModalProps {
+    open: boolean;
+    handleClose: () => void;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+    button: {
+        width: "90%",
+        marginLeft: "auto",
+        marginRight: "auto",
+        marginBottom: theme.spacing(5),
+    },
+    title: {
+        textAlign: "center",
+        marginTop: theme.spacing(4),
+        display: "block",
+    },
+    logo: {
+        margin: theme.spacing(2),
+    },
+
+}));
+
+function WelcomeModal(props: IWelcomeModalProps) {
+    const classes = useStyles();
+    const theme = useTheme();
+    const fullScreen = useMediaQuery(theme.breakpoints.down("sm"));
+
+    return (
+        <Dialog
+            fullScreen={fullScreen}
+            open={props.open}
+            onClose={props.handleClose}
+        >
+            <DialogContent>
+                <DialogContentText>
+                    <div>
+                        <div className={classes.logo}>
+                            <MULogo width="50%" height="100%" color="#83dffa"></MULogo>
```

Comment:
do you think we should extract the colour to a variable in constants.ts? could even modify the existing `LIGHTBLUE = "#4ED6E2"`